invoice/src/main/java/org/killbill/billing/invoice/generator/FixedAndRecurringInvoiceItemGenerator.java

```diff
@@ -239,7 +239,7 @@ List<InvoiceItem> processRecurringEvent(final UUID invoiceId, final UUID account
 
                 final RecurringInvoiceItemDataWithNextBillingCycleDate itemDataWithNextBillingCycleDate;
                 try {
-                    itemDataWithNextBillingCycleDate = generateInvoiceItemData(startDate, endDate, targetDate, billCycleDayLocal, billingPeriod, recurringBillingMode);
+                    itemDataWithNextBillingCycleDate = generateInvoiceItemData(startDate, endDate, targetDate, billCycleDayLocal, billingPeriod, recurringBillingMode, null);
```

Comment:
`internalCallContext` instead of `null`?

Reply:
Fixed in https://github.com/killbill/killbill/pull/1824/commits/038cb86de4ae59f86d385a7b11e090e3274a09c8